src/modules/workspace.ts

```diff
@@ -122,7 +122,15 @@ export class Workspace {
         this.files.writeFileSync(this.repos_md.value, reposList)
       }
 
-      this.files.writeFileSync(this.askpass_sh.value, `#!/bin/sh\n\necho '${token}'`)
+      const newToken = await token()
+      this.files.writeFileSync(this.askpass_sh.value, `#!/bin/sh\n\necho '${newToken}'`)
```

Comment:
**nitpick (non-blocking):** would you mind extracting this two lines into its own function and just called that function here and inside the `setInterval` block?

<sub>This comment follows the [**conventionalcomments.org**](https://conventionalcomments.org) standard</sub>